docs/source/en/model_doc/blip-2.md

```diff
@@ -61,6 +61,10 @@ If you're interested in submitting a resource to be included here, please feel f
 
 [[autodoc]] Blip2QFormerConfig
 
+## Blip2ModelWithoutLMConfig
```

Comment:
would be great to not add this config class. I'm ok with having separate `Blip2ForImageTextRetrieval` and `Blip2TextModelWithProjection` classes, but having a separate config is a bit weird. is there a reason there's a need for a separate config?

Reply:
Thanks for the feedback, I will remove Blip2ModelWithoutLMConfig and will try to update `Blip2Config` to support text_config=None, it was also recommended by  @younesbelkada in https://github.com/huggingface/transformers/pull/25612#discussion_r1364448600